content/foundations/color/base-scales.mdx

```diff
@@ -3,6 +3,73 @@ title: Base scales
 description:
   Primer base color scales makeup the foundation for all UI color considerations.
 ---
+## Neutral color scales
+
+<img
+  width="960"
+  alt="Two neutral scales stacked on top of each other labeled from 0 to 11. The top scale is the light scale, which starts with white and ends with black. The bottom scale is the dark scale, which starts with black and ends with white."
+  src="https://github.com/user-attachments/assets/3081e29d-6581-401b-91da-c014eee9cf66"
+/>
+
+Primer's neutral scales offer white, black, and shades of gray between 0 and 11. There are two versions of the neutral scale: light, and dark. 
+
+The light and dark scales are inverted respectively, with the light scale starting with white and the dark scale starting with black. By inverting the scales, light and dark themes are able to share many of the same functional color tokens without custom overrides.
+
+<img
+  width="960"
+  alt="A split view of how muted text colors reference the same base scale color in both light and dark themes."
+  src="https://github.com/user-attachments/assets/160e6384-3c33-4729-a6cf-96adce84f127"
+/>
+
+### Background
+
+The first five steps of the neutral scales are typically used for background colors. The two most commonly used background colors are `bgColor-default` and `bgColor-muted`. All contrast values for text and borders are calculated against `bgColor-muted` to ensure proper contrast ratios for both `muted` and `default`.
```

Comment:
Technically it's the first 6 (if you don't count white), but I knew what you meant.

Reply:
I had a shockingly hard time with numbers on this PR 😂